speedrunApiUtils.ts

```diff
@@ -56,16 +57,17 @@ export const ISO8601durationToString = (ISO8601duration: string) => {
   const result = [];
 
   ISO8601duration.split("").map((c) => {
-    if (c === "P" || c === "T" || c === "S") {
+    if (c === "P" || c === "T") {
     } else if (c === "H") {
       result.push(" hours ");
     } else if (c === "M") {
       result.push(" minutes ");
+    } else if (c === "S") {
+      result.push(" seconds");
```

Comment:
If a time had no seconds the previous version would post [blank seconds](https://twitter.com/bottest____/status/1544383134686449666). Changing this [here](https://twitter.com/bottest____/status/1544384003033153536) fixes this issue.

Reply:
Of course, good vision.